mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -452,29 +452,62 @@ LogicalResult PopulateParamsAccel::obtainTuningParameters(
 const InitParamsAccel
 PopulateParamsXDL::initParameters[PopulateParamsXDL::nInitParameters] = {
   // M/block N/block K/block M/wave N/wave kPack forceUnroll bCopyMore
+  {128, 256, 4, 64, 128, 4, true, true}, 
+  {128, 128, 8, 64, 64, 1, true, true},  
   {128, 128, 4, 64, 64, 4, true, true},
-  {64, 64, 8, 32, 32, 4, true, true},
-  {32, 64, 4, 32, 64, 4, true, true},
-  {32, 64, 2, 8, 64, 4, true, true},
-  {4, 64, 16, 4, 64, 1, true, true}
+  {128, 128, 2, 64, 64, 8, true, true}, 
+  {64, 256, 8, 64, 64, 1, true, true},   
+  {64, 128, 2, 32, 64, 4, true, true},   
+  {64, 64, 4, 32, 32, 4, true, true},    
+  {64, 64, 4, 32, 32, 8, true, true},   
+  {64, 64, 4, 16, 64, 4, true, true},   
+  {64, 64, 2, 32, 32, 8, true, true},   
+  {32, 64, 4, 32, 64, 4, true, true},   
+  {32, 64, 4, 8, 64, 4, true, true},   
+  {32, 64, 2, 32, 32, 8, true, true},   
+  {32, 32, 8, 16, 16, 4, true, true}, 
+  {32, 64, 2, 8, 64, 4, true, true},   
+  {16, 64, 8, 16, 16, 4, true, true}, 
+  {16, 32, 8, 16, 16, 8, true, true},  
+  {16, 16, 8, 16, 16, 8, true, true}, 
+  {8, 64, 8, 4, 64, 8, true, true}   
```

Comment:
I'd wait on @krzysz00's opinion but we'd likely to shoot to double the size of entries here for better chance of hitting a good perf_config combination. (Especially those parameters in the range of 64, 128.)